README.md

```diff
@@ -1,11 +1,11 @@
-# Nimiq Accounts Manager <!-- omit in toc -->
+# Nimiq Hub <!-- omit in toc -->
 
-The **Accounts Manager** (or **Nimiq Accounts**) provides a unified interface for
+The **Hub** provides a unified interface for
```

Comment:
This is a good time to remove the stars here, as they appear in link previews to NPM and look bad IMHO in there.
```suggestion
The Nimiq Hub provides a unified interface for
```